travis-ci.sh

```diff
@@ -31,7 +31,11 @@ if [ $e2eCode -ne 0 ]; then
     ./sonobuoy logs
     mkdir results; tar xzf $outFile -C results
     find results
-    find results/plugins -exec cat {} \;
+    find results/plugins -type f \
+      -exec echo {} \; \
```

Comment:
This could be replaced with the `-printf` flag to `find`.
```suggestion
      -printf '%p\n' \
```